apps/schedulii-ui/src/pages/Calendar.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  startOfMonth,
+  endOfMonth,
+  eachDayOfInterval,
+  format,
+  subDays,
+  addDays,
+  subMonths,
+  addMonths,
+  isSameDay,
+} from 'date-fns';
+
+import { useState } from 'react';
+
+const Calendar = () => {
+  // Get details about the current month
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const daysOfWeek = [
+    'Sunday',
```

Comment:
For a small scale widget, it might be better to just use the first letter of each day. Or some logic that doesn't let the days overflow into one another